test/index.test.ts

```diff
@@ -149,4 +149,16 @@ describe('fp-ts-ramda', () => {
       )
     );
   });
+
+  it('prop', () => {
+    fc.assert(
```

Comment:
@kightlingerh maybe we could increase coverage a bit by changing this test to something like:

```ts
  fc.assert(
      fc.property(
        fc
          .object()
          .filter(obj => Object.keys(obj).length > 0)
          .chain(obj =>
            fc.oneof(...Object.keys(obj).map(fc.constant)).map(existingKey => ({
              existingKey,
              objWithKey: obj
            }))
          ),
        ({ objWithKey, existingKey }) => {
          return (
            eqNumber.equals(R.prop(existingKey, objWithKey), FR.prop(existingKey, objWithKey)) &&
            eqNumber.equals(R.prop(existingKey)(objWithKey), FR.prop(existingKey)(objWithKey))
          );
        }
      )
    );
  });

  fc.assert(
    fc.property(fc.object(), fc.string(), (obj, key) => {
      return (
        eqNumber.equals(R.prop(key, obj), FR.prop(key, obj)) && eqNumber.equals(R.prop(key)(obj), FR.prop(key)(obj))
      );
    })
  );
```

Reply:
Looks good. I am happy to defer to you here, I'm not too familiar with `fast-check`. I made one small update where I use `strictEquals` instead of `eqNumber`.

Reply:
> I am happy to defer to you here

👍 I'll make this change soon

> I made one small update where I use strictEquals instead of eqNumber.

kind of related, I just found [this issue](https://github.com/dubzzz/fast-check/issues/435) with fast-check typings (my proposed change above shouldn't really compile 😅)